coalib/parsing/ConfParser.py

```diff
@@ -32,6 +32,23 @@ def __init__(self,
         self.__rand_helper = None
         self.__init_sections()
 
+    def check_valid_lines(self, lines):
+        """
+        This function checks for some invalid config file commands like
+        default_action: *: ShowPatchAction
+        which show error in the previous line. This function gives the
+        correct error and raises System Exit in this case.
```

Comment:
1.
```
System Exit
```
-->
```
``SystemExit``
```

Reply:
I don't raise a system exit now. so I will remove it